components/RegisterForm.vue

```diff
@@ -1,107 +1,119 @@
-      <div class="column q-pa-md">
-        <form class="full-width">
-          <q-input
-            v-model="name"
-            outlined
-            stack-label
-            hint="Please enter your name."
-            :error="vuelidate.name.$error"
-            error-message="Your name must be at least 1 character."
-            label="Name"
-            autofocus
-            bottom-slots
-            @blur="vuelidate.name.$touch" />
-          <q-input
-            v-model="ctrl.email"
-            outlined
-            stack-label
-            :hint="emailAvailable || 'Please enter your email address.'"
-            :error="vuelidate.ctrl.email.$error ||
-              (vuelidate.ctrl.email.$dirty && !isEmailUnique && !checkingEmail)"
-            :error-message="emailErrorMessage"
-            type="email"
-            label="Email"
-            class="q-mt-md"
-            bottom-slots
-            :loading="checkingEmail"
-            @blur="vuelidate.ctrl.email.$touch" />
-        </form>
-        <div class="row">
-          <q-checkbox
-            v-model="tosAgree"
-            dense />
-          <span class="q-my-md q-ml-sm">
-            I agree to the
-            <a
-              href=""
-              @click.stop.prevent="showTosModal = true">
-              Terms of Service
-            </a>
-          </span>
-        </div>
-        <q-dialog
-          v-model="showTosModal"
-          :maximized="maximizeModal">
-          <q-card
-            :style="!maximizeModal && {width: '700px', maxWidth: '80vw'}">
-            <q-toolbar class="bg-primary text-white">
-              <q-toolbar-title>Terms of Service</q-toolbar-title>
-              <q-btn
-                v-close-popup
-                dense
-                flat
-                label="Close">
-                <q-tooltip>Close</q-tooltip>
-              </q-btn>
-            </q-toolbar>
-            <q-card-section>
-              <p
-                v-for="(line, index) in tos"
-                :key="index">
-                {{line}}
-              </p>
-            </q-card-section>
-          </q-card>
-        </q-dialog>
-        <p class="row text-left">
-          When you click "Register", you may be asked to grant permission to
-          show your wallet in your browser's wallet selector. If you decline,
-          you can change this later under "Settings".
+  <q-card
+    class="full-width"
+    flat
+    bordered>
+    <q-card-section class="q-mt-lg text-center">
+      <img
+        v-if="branding.logo"
+        :src="branding.logo"
+        :style="`height: ${branding.logoSize.desktop}; filter:invert(1)`">
+    </q-card-section>
+    <q-card-section class="column text-center q-px-xl">
+      <form class="full-width">
+        <q-input
+          v-model="name"
+          outlined
+          stack-label
+          color="dark"
+          :error="vuelidate.name.$error"
+          error-message="Your name must be at least 1 character."
```

Comment:
This is equivalent to "Name is required"--which would be less awkwardly personal... We should do a language review and fix at least this one: https://github.com/digitalbazaar/bedrock-vue-wallet/issues/84